puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/deploy_yum.yaml

```diff
@@ -0,0 +1,33 @@
+- job:
+    name: deploy_yum
+    project-type: workflow
+    logrotate:
+      daysToKeep: 2
+      numToKeep: -1
+    triggers:
+      - timed: "" # ???
+    paramters:
+      - string:
+          name: os
+          default: el6
+          description: 'OS code, e.g. <code>el6</code>, <code>f18</code>'
+      - string:
+          name: repo_source
+          default: foreman-nightly
+          description: 'Repo on koji to test, e.g. <code>foreman-nightly</code> or <code>foreman-1.2</code>'
+      - string:
+          name: repo_dest
+          default: nightly
+          description: 'Relative path on yum.tf.org to sync to, e.g. <code>nightly</code> or <code>releases/1.2</code>'
+      - boolean:
+          name: merge
+          default: false
+          description: 'When true, will only add to and not delete the existing repo, then will run <code>createrepo</code> to merge the new packages into the existing repo.'
+      - string:
+          name: ruby
+          default: '2.0.0'
+          description: 'Version of Ruby to run deploy script under'
```

Comment:
This param is not used anywhere, and while it was on the original, I think we can just drop it.

Reply:
But the commands do use ```withRvm``` so either it needs to fully go to rvm or drop it.

Reply:
Currently withRVM defaults to use 2.0.0 so I think it can be dropped.